src/pass.h

```diff
@@ -104,22 +118,22 @@ class Pass : public WasmWalker {
 // e.g. through PassRunner::getLast
 
 // Handles names in a module, in particular adding names without duplicates
-class NameManager : public Pass {
+class NameManager : public WalkerPass<WasmWalker<NameManager, void> > {
```

Comment:
You can run `clang/tools/clang-format-diff.py` to only format what you've changed. I use the emacs integration, where formatting is just one key-combo away :)